common/static/common/templates/discussion/inline-discussion.underscore

```diff
@@ -13,6 +13,7 @@
             <div class="forum-nav-bar">
                 <button class="btn-link all-posts-btn">
                     <span class="icon fa fa-chevron-prev" aria-hidden="true"></span>
+                    <span class="sr-only"><%- gettext('Return to') %></span>
```

Comment:
Unfortunately concatenating strings like this won't work for localization if the text needs to appear in a different order. I think we probably need two full strings, the screen reader version and the non-screen version version.

I think there might also be a problem with the screen reader not seeing a space between 'to' and 'All'.